TYPO3.Media/Classes/TYPO3/Media/ViewHelpers/ImageViewHelper.php

```diff
@@ -18,12 +18,12 @@
 use TYPO3\Media\Domain\Model\ThumbnailConfiguration;
 
 /**
- * Renders an <img> HTML tag from a given TYPO3.Media's image instance
+ * Renders an <img> HTML tag from a given Neos.Media's image instance
  *
  * = Examples =
  *
  * <code title="Rendering an image as-is">
- * <typo3.media:image image="{imageObject}" alt="a sample image without scaling" />
+ * <neos.media:image image="{imageObject}" alt="a sample image without scaling" />
```

Comment:
Again: Even though this is just a comment (used to render documentation!), it will lead people to the use of a not (yet) existing ViewHelper. Repeat below. So, please only merge with the actual vendor name change.